src/applications/hca/config/chapters/veteranInformation/genderIdentificationInformation.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { genderLabels } from 'platform/static-data/labels';
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+import PrefillMessage from 'platform/forms/save-in-progress/PrefillMessage';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+const { gender } = fullSchemaHca.properties;
+
+const GenderIdentificationInfo = (
+  <>
+    <p style={{ color: 'gray' }}>
```

Comment:
Can you use a design system class here? https://design.va.gov/utilities/color

Reply:
I agree with @timwright12 need to use design system color in here on my other PR I used [vads-u-color--gray-medium](https://github.com/department-of-veterans-affairs/vets-website/pull/19337/files#diff-505884e19a88286371f1ab60c92d2ae27b4a3dd34d2d7ec6c2387b3419222430R28) here.

Reply:
Design class used. Thanks.